src/lib/stores/networksStore.ts

```diff
@@ -0,0 +1,54 @@
+import { writable, type Writable } from 'svelte/store';
+
+/**
+ * Represents the available networks.
+ */
+export enum Network {
+  NONE = 'NONE',
+  TESTNET = 'TESTNET',
+  MAINNET = 'MAINNET',
+  LOCALHOST = 'LOCALHOST',
+  CUSTOM = 'CUSTOM',
+}
+
+/**
+ * Represents information about a network.
+ */
+export interface NetworkInfo {
+  name: string | undefined;
+  endpoint: URL | undefined;
+  genesisHash: string | undefined;
+}
+
+/**
+ * Mapping of network types to network information.
+ */
+export const networkToInfo: Record<Network, NetworkInfo> = {
+  NONE: { name: undefined, endpoint: undefined, genesisHash: undefined },
+  TESTNET: {
+    name: 'TESTNET',
+    endpoint: new URL('wss://rpc.rococo.frequency.xyz'),
+    genesisHash: '0x0c33dfffa907de5683ae21cc6b4af899b5c4de83f3794ed75b2dc74e1b088e72',
+  },
+  MAINNET: {
+    name: 'MAINNET',
+    endpoint: new URL('wss://0.rpc.frequency.xyz'),
+    genesisHash: '0x4a587bf17a404e3572747add7aab7bbe56e805a5479c6c436f07f36fcc8d3ae1',
+  },
+  LOCALHOST: { name: 'LOCALHOST', endpoint: new URL('ws://127.0.0.1:9944'), genesisHash: undefined },
+  CUSTOM: { name: 'CUSTOM', endpoint: new URL('ws://127.0.0.1:9944'), genesisHash: undefined },
```

Comment:
Curious about why the custom endpoint is the same as localhost. As stated in your other PR, I assumed that endpoint would default to an empty string for custom and the user would have to enter a new endpoint in the input.

Reply:
I was trying to mimic the PolkadotJS UI:

<img width="322" alt="image" src="https://github.com/LibertyDSNP/provider-dashboard/assets/1135694/5e3963d9-e79b-43a2-b671-712d522857b1">

Reply:
We can follow up on this with design and product in the future. non-blocking.